apps/webpack.config.js

```diff
@@ -418,108 +418,108 @@ function createWebpackConfig({
+      //     //
+      //     // For more information see: https://webpack.js.org/guides/code-splitting/
+      //     'code-studio-multi': {
+      //       name: 'code-studio-common',
+      //       minChunks: Object.keys(appsEntries).length + 1,
+      //       chunks: chunk => {
+      //         const chunkNames = Object.keys(CODE_STUDIO_ENTRIES).concat(
+      //           Object.keys(appsEntries)
+      //         );
+      //         return chunkNames.includes(chunk.name);
+      //       },
+      //       priority: 20,
+      //     },
+      //     vendors: {
+      //       name: 'vendors',
+      //       priority: 30,
+      //       chunks: chunk => {
+      //         // all 'initial' chunks except OTHER_ENTRIES
+      //         const chunkNames = Object.keys({
+      //           ...appsEntries,
+      //           ...CODE_STUDIO_ENTRIES,
+      //           ...INTERNAL_ENTRIES,
+      //           ...PEGASUS_ENTRIES,
+      //           ...PROFESSIONAL_DEVELOPMENT_ENTRIES,
+      //           ...SHARED_ENTRIES,
+      //         });
 
-              return chunkNames.includes(chunk.name);
-            },
-            test(module) {
-              return [
-                '@babel/polyfill/noConflict',
-                'immutable',
-                'lodash',
-                'moment',
-                'pepjs',
-                'radium',
-                'react',
-                'react-dom',
-                'wgxpath',
-              ].some(libName =>
-                new RegExp(`/apps/node_modules/${libName}/`).test(
-                  module.resource
-                )
-              );
-            },
-          },
-          p5lab: {
-            name: 'p5-dependencies',
-            priority: 10,
-            minChunks: 2,
-            chunks: chunk =>
-              ['spritelab', 'gamelab', 'dance'].includes(chunk.name),
-            test: module => /p5/.test(module.resource),
-          },
-        },
-      },
+      //         return chunkNames.includes(chunk.name);
+      //       },
+      //       test(module) {
+      //         return [
+      //           '@babel/polyfill/noConflict',
+      //           'immutable',
+      //           'lodash',
+      //           'moment',
+      //           'pepjs',
+      //           'radium',
+      //           'react',
+      //           'react-dom',
+      //           'wgxpath',
+      //         ].some(libName =>
+      //           new RegExp(`/apps/node_modules/${libName}/`).test(
+      //             module.resource
+      //           )
+      //         );
+      //       },
+      //     },
+      //     p5lab: {
+      //       name: 'p5-dependencies',
+      //       priority: 10,
+      //       minChunks: 2,
+      //       chunks: chunk =>
+      //         ['spritelab', 'gamelab', 'dance'].includes(chunk.name),
+      //       test: module => /p5/.test(module.resource),
+      //     },
+      //   },
+      // },
```

Comment:
👋 Vijaya mentioned you were soliciting early feedback on this PR. does commenting out this block help local development times? my impression was that splitChunks only affected the optimized/production build.

Reply:
Thanks Dave. I updated the description with timing benchmarks - not sure whether something changed since we first added splitChunks, but at present, it does appear to affect the dev build.

Reply:
I see, nice find! enabling this only outside of DEV mode makes sense to me